tsconfig.json

```diff
@@ -2,6 +2,12 @@
 	"extends": "@tsconfig/node18-strictest-esm/tsconfig.json",
 	"compilerOptions": {
 		"outDir": "./dist",
+		"lib": [
+			"esnext"
+		],
+		"module": "esnext",
+		"target": "esnext",
```

Comment:
`"module": "esnext"` is required to import json files, which I find very handy. Updating also `lib` and `module` to stay consistent.